lib/themes.dart

```diff
@@ -0,0 +1,82 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+MaterialColor generateMaterialColor(Color color) {
+  return MaterialColor(color.value, {
+    50: tintColor(color, 0.5),
+    100: tintColor(color, 0.4),
+    200: tintColor(color, 0.3),
+    300: tintColor(color, 0.2),
+    400: tintColor(color, 0.1),
+    500: tintColor(color, 0),
+    600: tintColor(color, -0.1),
+    700: tintColor(color, -0.2),
+    800: tintColor(color, -0.3),
+    900: tintColor(color, -0.4),
+  });
+}
+
+int tintValue(int value, double factor) =>
+    max(0, min((value + ((255 - value) * factor)).round(), 255));
+
+Color tintColor(Color color, double factor) => Color.fromRGBO(
+    tintValue(color.red, factor),
+    tintValue(color.green, factor),
+    tintValue(color.blue, factor),
+    1);
+
+final primarySwatch = generateMaterialColor(Themes.accentColor);
+
+class Themes {
+  static const accentColor = Color(0xFF009FDF);
```

Comment:
I'm not so sure storing the color here is a good idea since it makes for two ways to access the same color. The ThemeData seems legit though.